### ISSUE TITLE
Shut down results in errors in vscode jupyter notebook

### ISSUE THREAD

**Issue by Fred-Wu** [man-group/dtale#786]
I am running dtale from a vscode jupyter notebook on my MacOS M1 . 

When I tried to shutdown an instance, the following error appeared in both cell output or browser no matter whether `open_browser` was set true or not. 

```

Internal Server Error

Error:

500 Internal Server Error: The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.

Stacktrace:

Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/envs/tensor-macos/lib/python3.11/site-packages/flask/app.py", line 2529, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/miniforge/base/envs/tensor-macos/lib/python3.11/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/miniforge/base/envs/tensor-macos/lib/python3.11/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/miniforge/base/envs/tensor-macos/lib/python3.11/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/miniforge/base/envs/tensor-macos/lib/python3.11/site-packages/dtale/app.py", line 440, in shutdown
    shutdown_server()
  File "/opt/homebrew/Caskroom/miniforge/base/envs/tensor-macos/lib/python3.11/site-packages/dtale/app.py", line 426, in shutdown_server
    raise RuntimeError("Not running with the Werkzeug Server")
RuntimeError: Not running with the Werkzeug Server


```

**Comment by Fred-Wu** [comment]
I am just using it through standard jupyter notebook in vscode, no specific server setting was made. 

<img width="744" alt="image" src="https://github.com/man-group/dtale/assets/4111978/59839bd1-730f-4a74-9a04-e55bde9f11b9">


Then shutting down results in 

<img width="761" alt="image" src="https://github.com/man-group/dtale/assets/4111978/d63e9d5a-c178-4cd5-bf7d-676a36e0a41e">

**Comment by aschonfeld** [comment]
@Fred-Wu so I found the issue.  Looks like the latest versions of Werkzeug have deprecated the `werkzeug.server.shutdown` function (as described [here](https://github.com/plotly/jupyter-dash/issues/63))

I'm going to continue digging to see if there is a different way to kill the server, but for now I think it should get killed when you end the jupyter notebook.

**Comment by aschonfeld** [comment]
@Fred-Wu I've fixed this issue in v3.5.0 of D-Tale. It's available on pypi and conda-forge. Let me know if you have any issues.

Also, if you haven't already, please put your ⭐ on the repo when you get a sec. Thanks! 🙏
